base/src/main/java/org/aya/concrete/parse/AyaParsing.java

```diff
@@ -76,7 +76,7 @@
     @NotNull Reporter reporter, @NotNull String text,
     @NotNull BiFunction<AyaProducer, AyaParser, T> tree
   ) {
-    var sourceFile = new SourceFile(Path.of("stdin"), text);
+    var sourceFile = new SourceFile("<stdin>", Path.of("stdin"), text);
```

Comment:
Please, don't do this. On Windows, it'll say that it's an invalid path.

Reply:
Not anymore. The display field is just a plain `String` now :)

Reply:
Noice.